python/train.py

```diff
@@ -115,56 +125,107 @@ def print_eval(params, hparams):
   Args:
     params: model params
     hparams: a tf.HParams object created by create_default_hparams().
+    model_dir: Name of model dir to store checkpoints.
+    eval_set: A list of evaluation data dictionaries generated by
+      'io.read_dataset_from_files'.
   """
-  (_, eval_input_fn, _, _, _) = \
-      ffcc_input.input_builder_stratified(DATA_DIR,
-                                          batch_size=1,
-                                          num_epochs=1,
-                                          bucket_size=1)
+  (_, eval_input_fn, _) = \
+      ffcc_input.input_builder_stratified(eval_set, batch_size=1,
+                                          num_epochs=1)
 
   run_config = tf.estimator.RunConfig()
   estimator = tf.estimator.Estimator(
       model_fn=model.model_builder(hparams),
       params=params,
-      model_dir=MODEL_DIR,
+      model_dir=model_dir,
       config=run_config)
 
   for result in estimator.predict(input_fn=eval_input_fn):
     burst_id = result['burst_id']
     uv = result['uv']
     # Convert UV to RGB illuminants for visualization
     with tf.Graph().as_default():
-      with tf.Session() as sess:
+      with tf.compat.v1.Session() as sess:
         uv_batch = uv.reshape((1, 2))
         rgb = sess.run(ops.uv_to_rgb(uv_batch))[0]
-    tf.logging.info('id=%s uv=%s rgb=%s', burst_id.decode('utf-8'), uv, rgb)
+    tf.compat.v1.logging.info('id=%s uv=%s rgb=%s', burst_id.decode('utf-8'),
+                              uv, rgb)
 
 
 def main(_):
   # Something about this code causes optimization to perform very poorly when
   # run on a GPU, so as a safeguard we prevent the user from using CUDA.
   # TODO(barron/yuntatsai): Track down the source of this discrepancy (maybe
   # something involving the FFT gradients?).
-  assert not tf.test.is_built_with_cuda()
 
-  # TODO(fbleibel): populate params and hparams based on project files.
-  hparams = {}
-  params = {}
+  assert not tf.test.is_built_with_cuda()
 
-  # TODO(fbleibel): load train and eval set from disk.
-  train_set = []
-  eval_set = []
+  if tf.io.gfile.exists(PROJECT_NAME + '_hyperparams.json'):
+    print('Loading hyperparameters from %s_hyperparams.json' % PROJECT_NAME)
+    with open(PROJECT_NAME + '_hyperparams.json', 'r') as hparams_file:
+      hparams = json.load(hparams_file)
+  else:
+    # TODO(fbleibel): populate params and hparams based on project files.
+    hparams = []
+    with open(PROJECT_NAME + '_hyperparams.json', 'w') as fp:
+      json.dump(hparams, fp)
+
+  if tf.io.gfile.exists(PROJECT_NAME + '_params.json'):
+    print('Loading hyperparameters from %s_params.json' % PROJECT_NAME)
+    with open(PROJECT_NAME + '_params.json', 'r') as params_file:
+      params = json.load(params_file)
+  else:
```

Comment:
Not sure if it's helpful to define a default set of parameters; it's simple to copy them from another project on disk to start a new project. I'd just error loudly (with an assert) if you can't find the params file.